src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -133,6 +135,72 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void shouldCreateDeepCopy() {
+        AccountAttributes account = createInvalidAccountAttributesObject();
```

Comment:
If you find the help methods are not useful (e.g. `createInvalidAccountAttributesObject`), you can always use `Builder` to build your own test data.